README.md

```diff
@@ -85,6 +85,7 @@ git push -f
 ```
 
 ### Docker Commands
+*Make sure that the NON POOLING URL is changed to db not localhost before running"
```

Comment:
Should add a section for seeding the prod DB

Reply:
resolved.